src/utils.rs

```diff
@@ -184,3 +192,22 @@ pub(crate) fn g1_to_uncompressed(g1: G1) -> Result<Vec<u8>> {
 
     Ok(result.to_vec())
 }
+
+pub fn format_pairing_check_values(
+    message: Vec<u8>,
+    signature: Vec<u8>,
+    public_key: Vec<u8>,
+) -> [([u8; 64], [u8; 128]); 2] {
```

Comment:
Same comment as @gluax 
```suggestion
) -> Resul<[([u8; 64], [u8; 128]); 2]> {
```